VideoSearchWithAutoMLVision/AutoML_Vision_Classifier_PowerSkill/powerskill/vi.py

```diff
@@ -0,0 +1,138 @@
+import logging
+
+import requests
+
+
+class VideoIndexerAPI():
+    def __init__(self, vi_api_key, vi_location, vi_account_id):
+        self.vi_api_key = vi_api_key
+        self.vi_location = vi_location
+        self.vi_account_id = vi_account_id
+        self.access_token = None
+
+    def get_access_token(self):
+        """
+        Here we get an access token for the video indexer instance
+        :return:
+        """
+        logging.info('Getting video indexer access token...')
+        headers = {
+            'Ocp-Apim-Subscription-Key': self.vi_api_key
+        }
+
+        params = {
+            'allowEdit': 'true'
+        }
+        access_token_req = requests.get(
+            'https://api.videoindexer.ai/auth/{loc}/Accounts/{acc_id}/AccessToken'.format(
+                loc=self.vi_location,
+                acc_id=self.vi_account_id
+            ),
+            params=params,
+            headers=headers
+        )
+
+        access_token = access_token_req.text[1:-1]
```

Comment:
Where do we handle access token expiry? (it is valid for 1h)

Reply:
At the moment we obtain a new access token per request